solutions/quickstart/variables.tf

```diff
@@ -71,7 +71,7 @@ variable "cluster_name" {
 variable "address_prefix" {
   description = "The IP range that will be defined for the VPC for a certain location. Use only with manual address prefixes."
   type        = string
-  default     = "10.245.0.0/24"
+  default     = "10.10.10.0/24"
```

Comment:
Why is the address prefix changed? This has induced the change because of which upgrade test has failed. Do we really need this change ?